samples/test/features/mfa-password-and-email.feature

```diff
@@ -9,8 +9,6 @@ Feature: Multi-Factor Authentication with Password and Email
     When she fills in her username
       And she fills in her correct password
       And she clicks Login
-    Then She sees a list of factors
```

Comment:
With "Remember last used factor" feature, if the user has only 1 factor, it will automatically be selected. So these steps won't be necessary i.e. to select email from list of factors